src/elements/hero/src/index.tsx

```diff
@@ -49,13 +53,26 @@ const Hero: React.FC<Props> = ({
     breadcrumbs &&
     React.cloneElement(breadcrumbs, { variant: breadcrumbVariant })
 
+  const getDeviceDisplay = () => {
+    if (fgImageOnMobile && fgImageOnTablet) {
+      return 'block'
+    } else if (!fgImageOnMobile && fgImageOnTablet) {
+      return ['none', 'block']
+    } else if (fgImageOnMobile && !fgImageOnTablet) {
+      return ['block', 'none', 'block']
+    }
+
+    return ['none', 'none', 'block']
```

Comment:
I would have done this like this, what do you think? Might not need `getDeviceDisplay` then

```suggestion
    return [
      fgImageOnMobile ? 'block' : 'none',
      fgImageOnTablet ? 'block' : 'none',
      'block'
    ]
```